.github/workflows/stale.yml

```diff
@@ -0,0 +1,59 @@
+# See https://github.com/marketplace/actions/close-stale-issues
+
+name: Mark stale issues and pull requests
+
+on:
+  schedule:
+  # run at 21:30 UTC daily
+  - cron: "30 21 * * *"
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/stale@v3
+      with:
+        repo-token: ${{ secrets.GITHUB_TOKEN }}
+
+        # NB: We start with very long duration while trimming existing issues,
+        # with the hope to reduce when/if we get better at keeping up with user support.
+
+        # The number of days old an issue can be before marking it stale.
+        days-before-stale: 180
+        # Number of days of inactivity before a stale issue is closed
+        days-before-close: 30
+
+        # If an issue/PR is assigned, trust the assignee to stay involved
+        # Can revisit if these get stale
+        exempt-all-assignees: true
+        # Issues with these labels will never be considered stale
+        exempt-issue-labels: "need: discussion,cleanup"
```

Comment:
Are these exact, or are they a match? We have labels more like this: `type: cleanup` I don't think these exempt labels would ever trigger on this project.

Reply:
I assume they are exact, we could check the specification. I'm on vacation at the moment, feel free to amend or add commits.